File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        arr=[]
        c=0
        for i in range(len(nums)):
            if nums[i] == 0:
                c+=1
            else:
                arr.append(nums[i])
        
        nums[:]=arr+c*[0]
            

        

File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int numsLength=nums.length;
        int i,index=0;
        for(i=0;i<numsLength;i++)
        {
            if(nums[i]!=0)
            {
                int temp=nums[index];
                nums[index]=nums[i];
                nums[i]=temp;
                index++;
            }
        }
        
    }
}